client/src/components/ModalComponent.js

```diff
@@ -14,7 +14,7 @@ const ModalComponent = (props) => {
 
   return (
     <div>
-      <Button color="light" onClick={toggle}>{buttonLabel}</Button>
+      <a className="nav-link active" onClick={toggle}>{buttonLabel}</a>
```

Comment:
Add an href to # here, it's a gotcha that anchors (a tags) require an href in React

Reply:
Done

Reply:
it didn't like the # I guess, you can convert it to a button